.drone.yml

```diff
@@ -36,6 +36,56 @@ steps:
       exclude:
       - pull_request
 
+- name: publish
+  image: maven:3.8-openjdk-17
+  depends_on:
+    - sonar
+  environment:
+    ARTIFACTORY_TOKEN:
+        from_secret: artifactory_token
+  commands:
+  - mvn -s ./jparest_settings.xml -f pom.xml install -B org.apache.maven.plugins:maven-deploy-plugin:2.8.2:deploy -DskipTests
```

Comment:
Is it ok to skip the test run here. Wouldn't we want to ensure they all pass first?

Reply:
The unit tests run under the test step. Didn't want to add to build time by running unit tests twice.

Reply:
Ah right, yep makes sense.